### PR TITLE
Full 2023 annual capability upload

### PR DESCRIPTION
**Pull request by passnip96**
Full annual capability doc upload. Additions to modelling, decision support and capability with a new advancements page & archive. For internal review.

### CODE REVIEW

File: modelling/ecological_mod/index.md [hunk 1]
```diff
@@ -23,44 +23,107 @@ ____
...
 # References
 
-Bozec, Y. M., K. Hock, R. A. B. Mason, M. E. Baird, C. Castro-Sanguino, S. A. Condie, M. Puotinen, A. Thompson, and Mumby, P. J., 2022. Cumulative impacts across Australia’s Great Barrier Reef: A mechanistic evaluation. bioRxiv:1–37. [Read Here](https://esajournals.onlinelibrary.wiley.com/doi/epdf/10.1002/ecm.1494)
+Bozec, Y.M., Hock, K., Mason, R.A.B., Baird, M.E.,Castro-Sanguino, C., Condie, S.A., Puotinen, M., Thompson, A., & Mumby, P.J. (2022). Cumulative impacts across Australia’s Great Barrier Reef: A mechanistic evaluation. bioRxiv:1–37. https://doi.org/10.1002/ecm.1494
+
+Condie, S. and Condie, R. (2016). Retention of plankton within ocean eddies. Global Ecology and Biogeography. 25(10):1264-1277.https://doi.org/10.1111/geb.12485
+
+Condie, S., Plagányi-Lloyd, E., Morello, B., Hock, K., Beeden, R. (2018). Great Barrier Reef recovery through multiple interventions. Conservation Biology. 32(6):1356-1367. https://doi.org/10.1111/cobi.13161
+
+Condie, S.A., & Porobic, J. (2023). Coral Community Network (CoCoNet) Model: User Guide &Technical
```
Comment (jyucsiro) [inline modelling/ecological_mod/index.md]: "&Technical" --> "& Technical" ?

File: modelling/environmental_mod/index.md [hunk 2]
```diff
@@ -20,59 +20,113 @@ parent: Modelling
...
+
+Roelfsema, C.M., Kovacs, E.M., Ortiz, J.C., Callaghan, D.P., Hock, K., Mathieu, M., Johansen, K., Mumby, P., Wettle, M., Ronan, M., Lundgren, P., Kennedy, E.V., Phinn, S.R. (2020). Habitat maps to enhance monitoring and management of the Great Barrier Reef. Coral Reefs. 39, pp.1039–1054. https://doi.org/10.1007/s00338-020-01929-3
 
-Delandmeter, P. and Van Sebille, E., 2019. The Parcels v2. 0 Lagrangian framework: new field interpolation schemes. Geoscientific Model Development, 12(8), pp.3571-3584. [Read Here](https://gmd.copernicus.org/articles/12/3571/2019/)
+Steven, A.D., Baird, M.E., Brinkman, R., Car, N.J., Cox, S.J., Herzfeld, M., Hodge, J., Jones, E., King, E., Margvelashvili, N. and Robillot, C. (2019). eReefs: An operational information system for managing the Great Barrier Reef. Journal of Operational Oceanography, 12(sup2), pp.S12-S28. https://doi.org/10.1080/1755876X.2019.1650589
```
Comment (jyucsiro) [inline modelling/environmental_mod/index.md]: i think you left some authors out. suggest adding "et al." at the end of the authors, or list all authors.

File: capability/index.md [hunk 3]
```diff
@@ -1,9 +1,12 @@
...
-has_children: 
+has_children: true
 ---
 
-# RRAP M&DS Capability 
\ No newline at end of file
+# Capability 
+This section details specific use cases of the M&DS capability. 
+
+The M&DS suites provides knowledge and advice across multiple study areas to inform and support strategic and tactical intervention decisions on The Reef.
```
Comment (jyucsiro) [inline capability/index.md]: Suggest "The Reef" -> "the Great Barrier Reef"?

File: capability/overview.md [hunk 4]
```diff
@@ -22,29 +22,21 @@ ____
 
 ## Our Mission
 
-Our Mission is to help environmental decision makers on the Reef explore and identify robust intervention options under uncertainty. We do this using integrated state-of-the-art environmental, ecological and economic models and decision-support processes.
+Our Mission is to help environmental decision makers on The Reef explore and identify robust intervention options under uncertainty. We do this using integrated state-of-the-art environmental, ecological and economic models and decision-support tools.
```
Comment (jyucsiro) [inline capability/overview.md]: see above

File: capability/overview.md [hunk 5]
```diff
@@ -22,29 +22,21 @@ ____
...
--   Support informed strategic and tactical intervention decision-making and management planning for Reef, people and industries
-
-## RRAP models and decision-support systems (MDS) inform strategic and tactical Reef interventions
-
-The MDS suite is a collaborative  set of environmental, ecological, and economic models working within a decision-support framework. Climate models force the environmental and ecological layers, and influence intervention scope and effectiveness. Uncertainty is propagated throughout the information supply chain.  Scenario analyses of ecological and economic outcomes feed back to inform which intervention strategies and tactics are effective, safe and robust.  ​
+We use the best scientific knowledge of global and regional environmental forcing in space and time, and the biological and ecological responses of corals, to inform actions that can sustain a resilient Reef in the face of climate change. ​
 
+We translate ecological information to value functions and through scenario analyses of outcomes, identify intervention options that represent viable and lasting economic and social outcomes for The Reef and peoples. ​
```
Comment (jyucsiro) [inline capability/overview.md]: see above

File: modelling/ecological_mod/index.md [hunk 6]
```diff
@@ -23,44 +23,107 @@ ____
...
+
+### Model Description
+
+C-scape is a modeling framework specifically designed to integrate information about the key processes that exhibit variability within coral reefs and exert an impact on the growth of coral populations. Its purpose is to forecast the dynamics of coral populations across reef complexes to inform the dynamic optimisation of intervention deployments in space and time. C~scape compliments the regional ecosystem models by identifying how variability in individual responses can affect the confidence in predicted intervention benefits at a regional scale, reducing 
+the risk of selecting suboptimal recommendations during the decision-making process.
+
+### Key Features
+
+- C-scape's adaptability extends to either a solitary reef or a cluster of small reefs covering a selected geographic region, typically spanning from 10 to 40 kilometers in width.
```
Comment (jyucsiro) [inline modelling/ecological_mod/index.md]: C~scape?

Comment (jyucsiro) [inline modelling/ecological_mod/index.md]: C~scape?

File: modelling/ecological_mod/index.md [hunk 7]
```diff
@@ -23,44 +23,107 @@ ____
...
 
 ## C~scape
-C-scape is a modeling framework specifically designed to integrate information about the key processes that exhibit variability within coral reefs and exert an impact on the growth of coral populations. Its purpose is to forecast the dynamics of coral populations across reef complexes. C-scape's adaptability extends to either a solitary reef or a cluster of small reefs covering a selected geographic region, typically spanning from 10 to 40 kilometers in width.
+
+### Model Description
+
+C-scape is a modeling framework specifically designed to integrate information about the key processes that exhibit variability within coral reefs and exert an impact on the growth of coral populations. Its purpose is to forecast the dynamics of coral populations across reef complexes to inform the dynamic optimisation of intervention deployments in space and time. C~scape compliments the regional ecosystem models by identifying how variability in individual responses can affect the confidence in predicted intervention benefits at a regional scale, reducing
```
Comment (jyucsiro) [inline modelling/ecological_mod/index.md]: C~scape?

**Review by jyucsiro — CHANGES_REQUESTED** [review]
Thanks for this. Looks good - just some minor revisions suggested.

File: modelling/ecological_mod/index.md (hunk 1 above)
Comment (passnip96) [reply modelling/ecological_mod/index.md]: updated

File: modelling/environmental_mod/index.md (hunk 2 above)
Comment (passnip96) [reply modelling/environmental_mod/index.md]: Confirmed - there are only two authors for reference.

File: capability/index.md (hunk 3 above)
Comment (passnip96) [reply capability/index.md]: Noted & updated

File: capability/overview.md (hunk 4 above)
Comment (passnip96) [reply capability/overview.md]: Updated

File: capability/overview.md (hunk 5 above)
Comment (passnip96) [reply capability/overview.md]: Updated

File: modelling/ecological_mod/index.md (hunk 6 above)
Comment (passnip96) [reply modelling/ecological_mod/index.md]: Updated

Comment (passnip96) [reply modelling/ecological_mod/index.md]: Updated

File: modelling/ecological_mod/index.md (hunk 7 above)
Comment (passnip96) [reply modelling/ecological_mod/index.md]: Updated

**Comment by passnip96** [comment]
Hi Jono, hopefully we have addressed your proposed revisions. We have added a new commit. Back to you.

File: modelling/environmental_mod/index.md (hunk 2 above)
Comment (jyucsiro) [reply modelling/environmental_mod/index.md]: No I mean the eReefs paper. There are 18 co-authors on the paper (including Sharon, Barbara and myself omitted). Either list all or add et al. to the end. thanks!

File: modelling/ecological_mod/index.md (hunk 1 above)
Comment (jyucsiro) [reply modelling/ecological_mod/index.md]: oh, it still is showing up as "&Technical"

**Comment by passnip96** [comment]
Hi Jono, i hope these changes are sufficient. Please let me know if there's anything i've missed. Thanks

**Review by jyucsiro — APPROVED** [review]
looks good!

**Pull request merged**